scripts/core/interactive-article-actions-panel/actions/send-to-tab.tsx

```diff
@@ -38,7 +38,7 @@ export class SendToTab extends React.PureComponent<IProps, IState> {
         super(props);
 
         this.availableDesks = sdApi.desks.getAllDesks()
-            .filter((desk) => desk.send_to_desk_allowed)
+            .filter((desk) => desk?.send_to_desk_not_allowed === false)
```

Comment:
first, you don't need to use optional chaining here, `desk` will always be an object. If you try accessing a non existing property of an object you simply get undefined, no error is being thrown - thus no need for optional chaining.

More importantly, your check should be `desk.send_to_desk_not_allowed !== true`, not `=== false`. You need to keep in mind that it would be `undefined` in some cases. Since we aren't enforcing it being mandatory, we'll not be using an upgrade script either